itests/org.openhab.binding.modbus.tests/src/main/java/org/openhab/binding/modbus/tests/ModbusDataHandlerTest.java

```diff
@@ -774,7 +777,7 @@ public void testRefreshOnData() throws InterruptedException {
                 builder -> builder.withConfiguration(dataConfig), bundleContext);
         assertThat(dataHandler.getThing().getStatus(), is(equalTo(ThingStatus.ONLINE)));
 
-        verify(comms, never()).submitOneTimePoll(request, null);
+        verify(comms, never()).submitOneTimePoll(request, notNull(), notNull());
```

Comment:
Gives me an error. When using matchers (`notNull`), all arguments must be matchers.
I propose:
```
        verify(comms, never()).submitOneTimePoll(eq(request), notNull(), notNull());
```